src/vs/workbench/contrib/editSessions/common/workspaceStateSync.ts

```diff
@@ -136,9 +137,9 @@ export class WorkspaceStateSynchroniser extends AbstractSynchroniser implements
 					// Run URI conversion on the stored state
 					replaceUris(value);
 					// Write the stored state to the storage service
-					this.storageService.store(key, value, StorageScope.WORKSPACE, StorageTarget.USER);
+					this.storageService.store(key, value, StorageScope.WORKSPACE, StorageTarget.USER, ValueChangeSource.EXTERNAL);
```

Comment:
Can we make a global pass around usages of `store` and find all other places that should adopt this? I am sure we have similar code for the global / profile state sync?

Reply:
There are 327 uses of `store` to review, what do you think of getting this initial adoption in for workspace state sync to unblock Connor? I'll file an issue to adopt `source` for any other areas.

Reply:
Yeah I was not suggesting to check all usages of `store` but specifically areas where we know some kind of sync is involved or roaming. So I would scope it to yours and Sandeeps usages for now.

Reply:
I opened a separate PR for this https://github.com/microsoft/vscode/pull/185810